components/tools/OmeroPy/src/omero/plugins/fs.py

```diff
@@ -702,6 +719,36 @@ def ls(self, args):
         for ofile in fileset.listFiles():
             print ofile.path + ofile.name
 
+    def logfile(self, args):
+        """Return the logfile associated with a fileset"""
+        client = self.ctx.conn(args)
+        query = client.sf.getQueryService()
+        log = get_logfile(query, args.fileset.id.val)
+        if log is not None:
+            if args.name:
+                self.ctx.out(log.path.val + log.name.val)
+            elif args.size:
+                self.ctx.out(log.size.val)
+            else:
+                target_file = str(args.filename)
+                try:
+                    if target_file == "-":
+                        client.download(log, filehandle=sys.stdout)
+                        sys.stdout.flush()
+                    else:
+                        client.download(log, target_file)
+                except ValidationException, ve:
+                    # This should effectively be handled by None being
+                    # returned from the logfile query above.
+                    self.ctx.die(115, "ValidationException: %s" % ve.message)
+                except ResourceError, re:
+                    # ID exists in DB, but not on FS
+                    self.ctx.die(116, "ResourceError: %s" % re.message)
+        else:
+            self.ctx.die(
+                117,
+                "Log file not accessible for Fileset:%s" % args.fileset.id.val)
```

Comment:
In this case I hadn't intended to have a space. The error message is reflecting the input/ouput format of `Object:ID`. Though maybe that means the message should be:

```
"Log file not accessible for: Fileset:%s" % args.fileset.id.val)
```

I do think though that it should be `Logfile` for consistency with the help text.

though that looks a bit ugly.